### PR TITLE
Add link to /wiki/CodeReviewComments

### PR DESCRIPTION
**Pull request by pconerly**
Add a link to what rules golint is based off of.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ golint prints out style mistakes.
 
 Golint differs from govet. Govet is concerned with correctness, whereas
 golint is concerned with coding style. Golint is in use at Google, and it
-seeks to match the accepted style of the open source Go project.
+seeks to match the accepted style of the open source Go project, [as described here](https://github.com/golang/go/wiki/CodeReviewComments).
```
Comment (dsymonds) [inline README.md]: That's not particularly accurate. That page isn't technically a style guide, but I think this new sentence implies that it is.

I'd be fine having this link in here (but as golang.org/wiki/CodeReviewComments), but it needs different prose.

Comment (pconerly) [inline README.md]: ok. Would you consider Effective Go to be the style guide?

Comment (dsymonds) [inline README.md]: No, I don't think there *is* an "official Go style guide". The existing sentence here is intentionally loose because of that.

Comment (pconerly) [inline README.md]: How about this sentence at the end of the paragraph?

```
Golinter's style guidelines often come from [Effective Go](https://golang.org/doc/effective_go.html) or [golang.org/wiki/CodeReviewComments](golang.org/wiki/CodeReviewComments).
```

 Looking at the errors generated from the linter, there are the two style guidelines that are roughly used.

Comment (dsymonds) [inline README.md]: That's not really a true statement either. I'd phrase things like:

Golint makes suggestions for many of the mechanically checkable items listed in
Effective Go and the CodeReviewComments wiki page.

**Comment by pconerly** [comment]
@dsymonds there ya go

File: README.md [hunk 2]
```diff
@@ -34,7 +34,10 @@ Do not treat its output as a gold standard. We will not be adding pragmas
 or other knobs to suppress specific warnings, so do not expect or require
 code to be completely "lint-free".
 In short, this tool is not, and will never be, trustworthy enough for its
-suggestions to be enforced automatically, for example as part of a build process.
+suggestions to be enforced automatically, for example as part of a build process.
```
Comment (dsymonds) [inline README.md]: remove trailing space

**Comment by dsymonds** [comment]
Thanks. This is looking fine.

I need you to sign the CLA though. https://cla.developers.google.com/clas

**Comment by pconerly** [comment]
Cool, I submitted one, I'll let you know when it's gone through.

**Comment by dsymonds** [comment]
I can see the CLA for your GitHub account, so LGTM.

**Pull request closed without merging**